notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        print("probml_utils module not found")
</pre><p>We should follow the same notation, like we are generally not writing print() here, instead it should be something like </p><p>try:</p><p>&nbsp;&nbsp;from probml_utils import savefig, latexify</p><p>except ModuleNotFoundError :</p><p>&nbsp;&nbsp;%pip install ....</p><p>   from probml_utils import savefig, latexify</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Okay noted.</p>